geode-core/src/main/java/org/apache/geode/internal/cache/EvictionAttributesImpl.java

```diff
@@ -40,7 +44,7 @@
  *
  * @since GemFire 5.0
  */
-public class EvictionAttributesImpl extends EvictionAttributes {
+public class EvictionAttributesImpl extends EvictionAttributes implements Serializable {
```

Comment:
EvictionAttributes is already Serializable

Reply:
Sure